templates/javascript/README.md

```diff
@@ -0,0 +1,17 @@
+# Generated Snips skill
+
+This is a generated javascript snips skill using the `snips-template` tool.
```

Comment:
let's call it an app ?

Reply:
To be honest I just copy pasted the python description with some renaming. 😉